src/Directory.Build.props

```diff
@@ -12,8 +12,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <ZebusVersion>3.5.0</ZebusVersion>
-    <ZebusContractsVersion>3.0.0</ZebusContractsVersion>
+    <ZebusVersion>3.5.1</ZebusVersion>
+    <ZebusContractsVersion>3.0.1</ZebusContractsVersion>
```

Comment:
`Zebus.Contracts` did *not* change (and it is almost never supposed to). Don't bump its version.